circuit_maintenance_parser/parser.py

```diff
@@ -226,3 +226,24 @@ def parser_hook(self, raw: bytes):
     def parse_csv(raw: bytes) -> List[Dict]:
         """Custom CSV parsing."""
         raise NotImplementedError
+
+
+class Text(Parser):
+    """Html parser."""
+
+    _data_types = ["text/plain"]
+
+    def parser_hook(self, raw: bytes):
+        """Execute parsing."""
+        result = []
+        soup = bs4.BeautifulSoup(quopri.decodestring(raw), features="lxml")
```

Comment:
does this look like a Html parser? maybe we should add the `text/plain` into that data_type. I see it too much HTML specific for a generic Text parser